utils/io/fileutils/files.go

```diff
@@ -708,3 +708,29 @@ func MoveFile(sourcePath, destPath string) (err error) {
 	err = os.Remove(sourcePath)
 	return errorutils.CheckError(err)
 }
+
+// RemoveDirContents removes the contents of the directory, without removing the directory itself.
+// If it encounters an error before removing all the files, it stops and returns that error.
+func RemoveDirContents(dirPath string) (err error) {
```

Comment:
RemoveDirContents --> RemoveDirContent

Reply:
From what I saw on the internet, for this meaning, "contents" is better than "content".